t/cow.t

```diff
@@ -109,3 +108,41 @@ plan tests => 42;
     my $clone = clone($hash);
     ok ref $clone, "clone success - no SEGV";
 }
+
+{
+    note "playing with the limit: cowrefcnt_max";
+
+    my $max = cowrefcnt_max();
+    ok $max, "we got a max";
+
+    cmp_ok $max, '>', 2, "this should be greater than 2 :-)" or die;
+
+    # first let's do a stop just before max
+    my $str = "abcd";
+    my @a;
+    push @a, $str for 1 .. ( $max - 3 );
+    is cowrefcnt($str), $max - 2, "we are at max-2";
+
+    # now increase to max
+    push @a, clone($str); # fails on travis ???
```

Comment:
what do you mean by "fails on travis"?

Reply:
ok need to remove the comment :-) I mean that the regular perl version I'm using is behaving differently with COWPV with the notion of immortal cows... 
travis was just the stack of regular Perl